app/validators/email_blacklist_validator.rb

```diff
@@ -0,0 +1,10 @@
+class EmailBlacklistValidator < ActiveModel::EachValidator
+  def validate_each(record,attribute,value)
+    if AdminBlacklistedEmail.is_blacklisted?(value)
+      record.errors[attribute] << (options[:message] || I18n.t('email_blacklisted', default: "has been blocked at the owner's request. That means it can't be used in guest comments. Please check the address to make sure it's yours to use and contact AO3 Support if you have any questions."))
```

Comment:
Using the default option for I18n.t will putting things in to the the app we plan to use translation more difficult. I am not convinced the key is in the format we would expect ? 

https://github.com/otwcode/otwarchive/pull/2281/files might be useful ( if 'email_blacklisted' is a full key then default could be used usefully )